alerter.cpp

```diff
@@ -1,31 +1,30 @@
 #include <iostream>
 #include <assert.h>
+#include "alerter_stub.hpp"
 
 int alertFailureCount = 0;
 
-int networkAlertStub(float celcius) {
-    std::cout << "ALERT: Temperature is " << celcius << " celcius.\n";
-    // Return 200 for ok
-    // Return 500 for not-ok
-    // stub always succeeds and returns 200
-    return 200;
+float farenheitToCelcius(float farenheit) {
+    return (farenheit - 32) * 5 / 9;
 }
-
 void alertInCelcius(float farenheit) {
-    float celcius = (farenheit - 32) * 5 / 9;
+    float celcius = farenheitToCelcius(farenheit);
     int returnCode = networkAlertStub(celcius);
     if (returnCode != 200) {
         // non-ok response is not an error! Issues happen in life!
         // let us keep a count of failures to report
         // However, this code doesn't count failures!
         // Add a test below to catch this bug. Alter the stub above, if needed.
-        alertFailureCount += 0;
+        alertFailureCount += 1;
     }
 }
 
 int main() {
     alertInCelcius(400.5);

```

Comment:
How does the user of this API realize that the input is in Fahrenheit, whereas the method name is Celsius.